stl/inc/yvals_core.h

```diff
@@ -1182,6 +1182,17 @@
 #define _CXX20_REMOVE_CISO646
 #endif // ^^^ warning disabled ^^^
 
+#if !defined(_SILENCE_NONFLOATING_COMPLEX_DEPRECATION_WARNING)
+#define _DEPRECATE_NONFLOATING_COMPLEX                                                 \
+    [[deprecated("warning STL4036: "                                                   \

```

Comment:
should be `STL4037`, `STL4036` exists at line 1177.

Also  need to update `// next warning number: STL4037` to ` // next warning number: STL4038`